datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/Constants.java

```diff
@@ -24,6 +24,7 @@ private Constants() {}
   public static final String PROPERTIES_SCHEMA_FILE = "properties.graphql";
   public static final String FORMS_SCHEMA_FILE = "forms.graphql";
   public static final String ASSERTIONS_SCHEMA_FILE = "assertions.graphql";
+  public static final String COMMON_SCHEMA_FILE = "common.graphql";
```

Comment:
one thing that's missing is adding the new `common.graphql` file on the backend to our docs website auto-generation list. So we can add the following line to `docs-website/graphql/generateGraphQLSchema.sh`:
```
cat ../../datahub-graphql-core/src/main/resources/common.graphql >> combined.graphql
```